Data/Tools/ezEditor/VisualShader/Textures.ddl

```diff
@@ -342,7 +342,14 @@ SamplerState $prop0_AutoSampler;
     string %Type { "float" }
     string %DefaultValue { "1" }
   }
-  
+
```

Comment:
@C-Core I wanted to make a material that uses the pattern 3-way mapping in object space. So that the pattern doesn't change when moving it. I failed quite miserably, and think this is currently not possible, but don't understand why. The necessary Visual Shader nodes exist, but don't seem to work properly. I kept this change in, because I think it is generally correct, but it would be helpful, if you could have a look some time, whether this is really not possible.